angularjs-portal-frame/src/main/webapp/index.html

```diff
@@ -18,6 +18,16 @@
 </head>
 
 <body ng-controller="MainController as mainCtrl">
+<noscript>
+    <div class="alert alert-warning alert-dismissible" role="alert" style="margin-bottom:0;">
+    <div class="container">
+      <i class="fa fa-2x fw fa-exclamation-triangle pull-left"></i>
+      <strong>
+        Please <a href="http://enable-javascript.com" target="_blank">enable Javascript</a> to interact with all forms and features on our website.  For further assistance, contact the <a href="http://kb.doit.wisc.edu/helpdesk" target="_blank">DoIT Help Desk</a>.
```

Comment:
`http://kb.doit.wisc.edu/helpdesk` doesn't work for me.  I get a blank screen, under both [Chrome](http://cl.ly/image/1N0P2J3J1W2P) and [Safari](http://cl.ly/image/0E1u391t2z2M).